packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx

```diff
@@ -355,7 +360,7 @@ export class PaginationBar {
 
   private setPaginationBarContent = (): void => {
     const displayedItemsPerPageOptions =
-      this.itemsPerPageOptions?.slice(0, 3) ||
+      this.itemsPerPageOptions?.slice(0, this.itemsPerPageOptions.length) ||
```

Comment:
not sure the .slice is necessary now, if it should just show all the options passed in?